### ISSUE TITLE
Homepage on Android - issue with keyboard blocking Current Location

### ISSUE THREAD

**Issue by lizagarrison** [hackforla/food-oasis#587]
### Overview
When Android users go to the FOLA homepage and clicks in the "search by address, etc" field to enter their location, their keyboard appears and blocks the "Current Location" prompt, which makes it difficult to find. Once the user searches by current location, their keyboard remains up on the map/results page and blocks the map view.

[See screen recording demonstrating issue](https://drive.google.com/file/d/1yUouARdZr8VNXZs_QheqF7iN0DYOaY7W/view?usp=sharing)

**Comment by entrotech** [comment]
I'm using  a Pixel 3 with Android 10, and I'm not able to reproduce this issue.

https://drive.google.com/drive/u/0/folders/15uroQk6gYLDEaCSDRbTz9b_MIpNT6pYE

**Comment by entrotech** [comment]
In meeting on 7/14, we decided that the main problem is that there is a gap between the location text box and the drop-down list of choices under some circumstances. This is just more troublesome on a small android device where the drop-down list might be obscured by the deyboard drawer that is open when the user types in the location text box, so they won't see that the list is there.  I  closed this gap, which should help.

I also moved the subtitle "Your free food directory" back to where it belongs and removed the period at the end.

![image](https://user-images.githubusercontent.com/9939032/87613266-a5bf8000-c6c1-11ea-9a8b-b88b5c0d3526.png)
